ansible/monitoring.yml

```diff
@@ -55,6 +56,7 @@
         - "region:{{ region|default('eu-east') }}"
         - "env:{{ env|default('unknown') }}"
         - "color:{{ color|default('unknown') }}"
+        - "kind:{{ kind|default('peer') }}"
```

Comment:
I see in combination with already-merged https://github.com/aeternity/infrastructure/pull/84/files this delivers https://www.pivotaltracker.com/n/projects/2152594/stories/159620185